client/src/utils/ssotToBpmnParsing/ssotToBpmnParsing.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @category Client
+ * @module
+ */
+
+const DEFAULT_STARTER = 'Event_13c1i8b';
+const DEFAULT_PARENT = 'Participant_0cyhvx8';
+const DEFAULT_SPACING = '150,0';
+const DEFAULT_STARTEVENT_POSITION = '200,330';
+
+/**
+ * @description Will search through the ssot and return the only element without a predecessor
+ * @param {Object} ssot The ssot to process
+ */
+const findStartElement = (ssot) => (
+    ssot.elements.find((element) => (element.type === 'MARKER' && element.predecessorIds.length < 1)
+    ));
+
+/**
+ * @description Will create an order within the elements based on their positioning in the flow
+ * @param {Object} ssot The ssot to process
+ * @returns {Array} The elements of the ssot in the correct order in which they appear in the flow
+ */
+const buildCorrectOrder = (ssot) => {
+    const startElement = findStartElement(ssot);
+    const elementsInOrder = [startElement];
+    let currentElement = startElement;
+    for (let i = 1; i < ssot.elements.length; i++) {
+        currentElement = ssot.elements.find((element) => (element.id === currentElement.successorIds[0]));
+        elementsInOrder.push(currentElement);
```

Comment:
Not a must have but current.elements.foreach(....) would probably be nicer

Reply:
As we are not doing something for each element of the list, but only do it as often, as the list has elements I would prefer this style here to avoid confusion.